Android/build.gradle

```diff
@@ -13,7 +13,7 @@ dependencies {
     compile 'com.android.support:cardview-v7:22.1.0'
     compile 'com.android.support:recyclerview-v7:22.1.0'
 
-    compile 'com.o3dr.android:dronekit-android:2.3.32'
+    compile 'com.o3dr.android:dronekit-android:2.3.33'
```

Comment:
upgrade to version `2.3.34`. Version `2.3.33` has a bug that the newer version fixes.